### PR TITLE
Added html for section 2

### LINKED ISSUE #4: html section 2
We need to create the layout of the second section.

### PR DESCRIPTION
**Pull request by Tumakoff**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,43 +1,61 @@
...
+  <section class="download" id="download">
+    <div class="download__wrapper">
+      <div class="download__text-block">
+        <h1 class="download__title">Peaceful shopping mindful money</h1>
+        <p class="download__copy">
+          See how we can help with making your shopping experience and money management more ZEN.
+        </p>
+        <div class="download__links">
+          <a class="download__link" href="https://apple.com">
+            <img src="./assets/icons/apple.svg" alt="Apple icon">
```
Comment (Maiia2023) [inline index.html]: it is recommended to close single tag

File: index.html [hunk 2]
```diff
@@ -1,43 +1,61 @@
...
+          </a>
+          <a class="download__link" href="https://google.com">
+            <img src="./assets/icons/google.svg" alt="Google icon">
```
Comment (Maiia2023) [inline index.html]: it is recommended to close single tag

File: index.html [hunk 3]
```diff
@@ -1,43 +1,61 @@
...
-    <section>Секция 2</section>
-    <!--Секция 3-->
-    <section>Секция 3</section>
-    <!--Секция 4-->
-    <section>Секция 4</section>
-    <!--Подвал сайта-->
-    <footer>Подвал сайта</footer>
-  </body>
+      <div class="download__image-block">
+        <img src="./assets/images/1.jpg" alt="Bag and device">
```
Comment (Maiia2023) [inline index.html]: it is recommended to close single tag

File: index.html [hunk 4]
```diff
@@ -1,43 +1,61 @@
...
+          ZEN cards have a very neat feature that will help you whenever your electronic devices start failing after the
+          standard warranty ends, which is pretty common.
+        </p>
+        <p class="warranty__copy">
+          When you buy any electronics with your ZEN shopping Mastercard, their warranty gets extended by one year. How
+          does it work? It simply does. Automatically.
+        </p>
+      </div>
+      <div class="warranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket">
```
Comment (Maiia2023) [inline index.html]: it is recommended to close single tag

**Review by Maiia2023 — APPROVED** [review]

File: index.html
Comment (nagorapro) [inline index.html:1]: Don't use the formatter yet

**Review by nagorapro — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Good

**Pull request merged**